GANDLF/entrypoints/debug_info.py

```diff
@@ -37,7 +50,22 @@ def new_way():
     + "`gandlf_debugInfo` script would be deprecated soon."
 )
 def old_way():
-    _debug_info()
+    parser = argparse.ArgumentParser(
+        prog="GANDLF_DebugInfo",
+        formatter_class=argparse.RawTextHelpFormatter,
+        description="Generate debugging information for maintainers.\n\n"
+        + copyrightMessage,
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        metavar="",
```

Comment:
Hi @scap3yvt ! The error is raised here because of `metavar` param is passed (and somewhere in deep of ArgumentParser is does not expect `metavar` arg). I don't know why it fails; according to the [doc](https://docs.python.org/3.9/library/argparse.html#metavar) it should work properly. But, as this field is not critical for us, I propose just to remove this line. The code works properly without `metavar`

Reply:
Thanks!